tests/console/snapper_cleanup.pm

```diff
@@ -48,7 +48,18 @@ sub snapper_cleanup {
     for (1 .. $scratch_size_gb) { assert_script_run("$snap_create", 500); }
     assert_script_run('sync');
     script_run "echo There are `$snaps_numb` snapshots BEFORE cleanup";
-    assert_script_run("snapper cleanup number", 300);    # cleanup created snapshots
+    # cleanup created snapshots
+    # softfail the test if hitting bsc#1206814, so that the subsequent test runs are not affected
+    my $output = script_output('snapper cleanup number', 300, proceed_on_failure => 1);
```

Comment:
Careful!! `script_output` only redirects stdout, not stderr. If we're using `$output ne ''` later on and there would be an error written to stderr (which it should be) we are not detecting the error!

A quick solution would be to do the following:

```suggestion
    my $output = script_output('snapper cleanup number || echo "error detected"', 300, proceed_on_failure => 1);
```

That prints "error detected" in all error cases to stdout and we can check for this string in line 54.

Reply:
Hmm
`
proceed_on_failure - allows to proceed with validation when $script is failing (return non-zero exit code)
`

Reply:
Based on the VRs, the codes worked fine for both passed and failed tests :)

Reply:
Ah, I got the `if` block wrong, sorry. But still, we would not detect a failure where something is written and then `snapper` dies.